pkg/generate/testdata/models/apis/codedeploy/0000-00-00/generator.yaml

```diff
@@ -7,10 +7,13 @@ resources:
     # below, we're testing printer columns end up sorted properly in the CRD
     fields:
       DeploymentGroupName:
-        is_printable: true
+        print:
+          name: DeploymentGroup
       ApplicationName:
-        is_printable: true
+        print:
+          name: Application
       DeploymentConfigName:
-        is_printable: true
+        print:
+          name: DeploymentConfig
       Description:
-        is_printable: true
+        print: {}
```

Comment:
I'm not a huge fan of this, as I feel it is overly verbose and easy to mess up :)

How about we just remove the `IsPrintable` field and keep the `PrintName` field, but make it a `*string` instead of `string` and just require any printable column to have `PrintName` set to a string value (even if that string value is the same name as the field).

Reply:
The problem is that there other configuration we want to add the `PrinterColumn` generator. Like `order`, `priority`, `type` and probably more in the future.

So long term, instead of having something like:
```yaml
resources:
  Deployment:
    fields:
      DeploymentGroupName:
        is_printable: true
        print_name: DeploymentGroup
        print_order: 3
        print_priority: 1
      DeploymentConfigName:
        is_printable: true
        print_name: DeploymentConfig
        print_order: 2
        print_priority: 1
        print_type: string

```

we can have something like:

```yaml
resources:
  Deployment:
    fields:
      DeploymentGroupName:
        print:
          name: DeploymentGroup
          order: 3
          priority: 1
      DeploymentConfigName:
        print:
          name: DeploymentConfig
          order: 2
          priority: 1
          type: string
```

The second option way less verbose and easier to clearer to read/understand IMO.

Reply:
OK, you convinced me. :)